src/Appwrite/Utopia/Response/Model/Membership.php

```diff
@@ -80,7 +80,7 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'User list of roles',
                 'default' => [],
-                'example' => 'admin',
+                'example' => ['owner'],
```

Comment:
If we use array=true one line below do we still need to provide the example like this? do we have issues in other places as well?

Reply:
Yes, we need to. Our [membership docs](https://appwrite.io/docs/models/membership) shows an incorrect example:

![image](https://github.com/appwrite/appwrite/assets/1477010/cbc5fdd1-dcea-422e-b38f-2f8c9f3b3645)